tests/fixtures/direct_award_project_searches_fixture.json

```diff
@@ -0,0 +1,31 @@
+{
+  "searches": [
+    {
+      "active": true,
+      "createdAt": "2017-08-29T07:00:02.000000Z",
+      "createdBy": 10000,
+      "id": 1,
+      "projectId": 1,
+      "searchUrl": "http://search-api.digitalmarketplace.service.gov.uk/g-cloud/services/search?q=accelerator",
```

Comment:
Feels like the `searchUrl`s here don't fully exercise all the constructing/deconstructing of URLs that happens within the `view_project` route.  It feels quite complex and brittle, so would be nice to see tests that cover a wider range of search URLs with various odd combinations of filters, etc. and see them reconstructed into the correct frontend search URL.

I know that's not what you want to hear right now, but if it doesn't happen now it might never happen...

Reply:
Good idea.